lava-slave-install.md

```diff
@@ -0,0 +1,93 @@
+This is an howto install a lava-docker slave
+This howto is based on a Debian host.
+
+# Create a dedicated user
+The user will be lavadocker in our example. It must be in the docker group for doing docker operations.
+```
+useradd -G docker lavadocker
+```
+
+# Install all pre-requisites packages
+docker-compose need to be installed from PIP
+```
+apt-get install git python-pip
+```
+
+# Install docker-ce
+See https://docs.docker.com/install/linux/docker-ce/debian/ for more detailled informations.
+
+# install docker-compose for our user
+
+As lavadocker run:
+```
+pip install --user docker-compose
+```
+
+# Install a second network interface
+For our example it will be named enx0
+
+# Install a dhcpd listenning on enx0
+You have many choices, for our examples we will use isc-dhcp-server:
+```
+apt-get install isc-dhcp-server
+```
+
+## Configure the DHCPD server
+```
+sed -i 's,INTERFACESv4="",INTERFACESv4=""yourdedicatedinterfacename,' /etc/default/isc-dhcp-server
+```
+
+Add the following to /etc/dhcp/dhcpd.conf
+```
+subnet 192.168.66.0 netmask 255.255.255.0 {
+  range 192.168.66.11 192.168.66.250;
+	option routers 192.168.66.1;
+}
+```
+
+# Checkout lava-docker sources
+As lavadocker run:
+```
+git clone https://github.com/kernelci/lava-docker.git
+```
+
+# Create your boards.yaml
+## Get necessary values from master
+* A remote username (remote_user)
+* A remote token for this username (remote_user_token)
+* The FQDN for connecting to the master (remote_master)
+
+## Optional ZMQ keys
+If the master is using ZMQ encryption you need to:
+### Generate ZMQ keys
+Generate thoses keys via:
+```
+zmqauth/zmq_auth_gen/create_certificate.py --directory . nameofyourslave
+```
+This will produce two files:
+* A public key ending with ".key" (zmq_auth_key)
+* A private key ending with ".key_secret (zmq_auth_key_secret)
+
+### slave public key
+Give the slave public key to the master's admin
+
+### master public key
+Get the master public key from the master's admin and set it as "zmq_auth_master_key"
+Note: you dont need master private key (or need to touch a fake one).
+
+# Generate files
+As lavadocker run:
+```
+./lavalab-gen.py
+```
+
+# Run deploy.sh
+```
+cd output/`host`/
```

Comment:
You should provide an example `standalone-slave.yaml` file that has an example hostname, and then use that here instead of ``host``